### PR TITLE
Vet clinic: Query and update

### PR DESCRIPTION
**Pull request by manzitresor**
### Implemented Feature
update
groupby
transaction1
deleteTransaction
savepoint

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -4,4 +4,11 @@ INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)VALUE
...
 INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)VALUES ('Pikachu', '2021-01-07', 1, false, 15.04);
 INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)VALUES ('Devimon', '2017-05-12', 5, true, 11.0);
+INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES('Charmander','2020-02-08',0,false,-11);
+INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES('Plantmon','2021-11-15',2,true,-5.7);
+INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES('Squirtle','1993-04-02',3,false,-12.13);
+INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES('Angemon','2005-06-12',1,true,-45);
+INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES('Boarmon','2005-06-07',7,true,20.4);
+INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES('Blosson','1998-10-13',3,false,7);
+INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES('Ditto','2022-05-14',4,true,4);
```
Comment (AbelG101) [inline data.sql]: - [ ] Please make sure to insert the proper data as there's a typo in the name `Blosson` It should be `Blossom` the animal has also tried to escape so it should be `true` and it weighs `17` kgs not `7`
- [ ] The animal "Ditto" weighs `22.0`kgs so please fix this as well.

File: queries.sql [hunk 2]
```diff
@@ -7,4 +7,37 @@ SELECT date_of_birth from animals WHERE name IN ('Agumon', 'Pikachu');
...
+UPDATE animals SET species='unspecified';
+SELECT *FROM animals;
+ROLLBACK;
+SELECT *FROM animals;
+
+BEGIN TRANSACTION;
+UPDATE animals SET species='digimon' WHERE name LIKE '%mon';
+SELECT *FROM animals;
+UPDATE animals SET species='pokemon' where species IS NULL;
+SELECT *FROM animals;
```
Comment (AbelG101) [inline queries.sql:23]: - [ ] Please make sure to add a `commit` after making the updates to the animals and also make sure to verify the changes before and after adding the commit.

File: queries.sql [hunk 3]
```diff
@@ -7,4 +7,37 @@ SELECT date_of_birth from animals WHERE name IN ('Agumon', 'Pikachu');
...
+ BEGIN TRANSACTION;
+ DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+ SAVEPOINT del_1;
+ UPDATE animals SET weight_kg = weight_kg * -1;
+ ROLLBACK TO del_1;
+ UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+ COMMIT;
+
+ SELECT COUNT(*) FROM animals;
+ SELECT COUNT(*) FROM animals WHERE escape_attempts > 0;
```
Comment (AbelG101) [inline queries.sql]: - [ ] Ouch! it seems you've misinterpreted the question for this query as it asks "How many animals have never tried to escape?" but you're retrieving the animals that have tried to escape, so please reevaluate it.

File: data.sql
Comment (AbelG101) [inline data.sql:1]: - [ ] And finally please update your screenshots accordingly with the changes requested here.

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @manzitresor 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Species column is properly added ✔
- Proper transaction management ✔
- Correct use of Git Flow ✔
- Descriptive PR title with a great screenshot summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by manzitresor** [comment]
groupby
blossom
ditto

**Review by uwadonat — APPROVED** [review]
## Hi @manzitresor,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### Highlights
- Well organized and clean project ✔️
- Detailed pull request message with screenshot ✔️
- Professional readme file ✔️

ok

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**